charts/values/tigera-operator.yaml

```diff
@@ -0,0 +1,29 @@
+imagePullSecrets: {}
+resources: {}
+
+installation:
+  enabled: false
+  kubernetesProvider: ""
+
+apiServer:
+  enabled: false
+
+certs:
+  node:
+    key:
+    cert:
+    commonName:
+  typha:
+    key:
+    cert:
+    commonName:
+    caBundle:
+
+# Configuration for the tigera operator
+tigeraOperator:
+  image: tigera/operator
+  version: master
```

Comment:
Note to self that we either need to update this as part of the release or come up with something smarter (hopefully the latter)